lib/translator/inquiry.cc

```diff
@@ -187,6 +187,64 @@ void TranslateUnitSerialNumberVpd(
   memcpy(&buffer[sizeof(result)], &product_serial_number, result.page_length);
 }
 
+void TranslateBlockLimitsVpd(
+    const nvme_defs::IdentifyControllerData &identify_controller_data,
+    absl::Span<uint8_t> buffer) {
+  /*
+  The value is in units of the minimum memory
+  page size (CAP.MPSMIN) and is reported as a power of two (2^n).
+  A value of 0h indicates that there is no maximum data transfer size
+  */
+  if (identify_controller_data.mdts > 64) {
+    DebugLog("max transfer length is > 2^64");
+  }
+  uint32_t max_transfer_length =
+      identify_controller_data.mdts ? 1 << identify_controller_data.mdts : 0;
+
+  scsi_defs::BlockLimitsVpd result = {
+      .page_code = scsi_defs::PageCode::kBlockLimitsVpd,
+      .page_length = 0x003c,
```

Comment:
Can we add a named variable for `0x003c `?

Reply:
will make all the constants when we've consolidated the inquiry branches

Reply:
Best practice to add a TODO in that case